styles.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+    font-family: "Figtree-ExtraBold";
+    src: url('./assets/fonts/static/Figtree-ExtraBold.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: "Figtree-Medium";
+    src: url('./assets/fonts/static/Figtree-Medium.ttf') format('truetype');
+}
+
+body {
+    background-color: rgb(244, 208, 78);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+img {
+    border-radius: 10px;
+}
+
+h1, h2, p {
+    line-height: 150%;
+    letter-spacing: 0px;
+}
+
+h1 {
+    font-size: 24px;
+    font-family: "Figtree-ExtraBold", sans-serif;
+}
+
+h2 {
+    font-family: "Figtree-ExtraBold", sans-serif;
+}
+
+h2, p {
+    font-size: 16px;
+}
+
+p {
+    font-family: "Figtree-Medium", sans-serif;
+}
+
+.author {
+    display: flex;
+    align-items: center;
+}
+
+.tab {
+    display: flex;
+    padding: 0px;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.learning {
+    background-color: rgb(244, 208, 78);
+    border-radius: 5px;
+    padding: 5px 15px;
+    margin-bottom: 0;
+    font-size: 16px;
+}
+
+.wrapper-tab {
+    width: 384px;
+    margin: auto;
+    border: 1px solid black;
+    border-radius: 20px;
+    box-sizing: border-box;
+    padding: 24px;
+    background-color: white;
+    box-shadow: 7px 7px 0px black;
+}
+
+.avatar {
+    border-radius: 100%;
+    width: 32px;
+    height: 32px;
```

Comment:
I've mentioned not using fixed `width` or `height` so much for larger components. This is an instance (and images in general) where I think it's fine though 👍 If an image or some decorative element of the design needs a fixed `width` or `height`, then I think it makes sense to use the fixed version rather than `max-`